debian/control

```diff
@@ -15,6 +15,7 @@ Build-Depends: autoconf,
                default-jdk,
                git,
 	       gdisk,
+               grep,
```

Comment:
Can you ever assume a package is installed? Figured better to be safe than sorry.